aws-samples/amazon-chime-sdk
# Add meeting status notifier to React component lib meeting demo

This will help in testing various meeting statuses from Amazon Chime SDK component library for React.

**Issue #:**

**Description of changes:**
- Add a new component `MeetingStatusNotifier` to throw UI notifications based on meeting status updates from underlying Amazon Chime SDK component library for React.
- Update `amazon-chime-sdk-component-library-react` submodule which includes the new `MeetingStatus.Reconnecting` status.

**Testing**

1. How did you test these changes?

    1. Run the demo app.
    2. Verify correct connecting and connected statuses are shown in meeting screen.
    3. Turn internet off and verify reconnecting status is notified.
    4. Keep the internet off and see reconnecting statues comes up until the internet is connected back.
    5. Once internet is connected back verify meeting connected status is shown.
    6. Do the same again, but, this time do not turn ON the internet and verify meeting failed message is notified and you are re-directed to home page.

3. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?
7. If applicable, have you run `npm run build` successfully locally to fix all warnings and errors?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
